draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -346,18 +346,18 @@
       <t>
         This section describes restrictions associated with the usage of
         SDP parameters within a BUNDLE group. It also describes, when parameter
-        and attribute values have been associated with each bundled "m=" line, how to
-        calculate a value for the whole BUNDLE group.
+        and attribute values are present in multiple bundled "m=" sections, how to
+        choose a value for the BUNDLE transport.
       </t>
     <section title="Connection Data (c=)" anchor="sec-sdp-cons-c" toc="default">
       <t>
         The "c=" line nettype value <xref format="default" pageno="false"
-        target="RFC4566"/> associated with a bundled "m=" line MUST be 'IN'.
+        target="RFC4566"/> in a bundled "m=" section MUST be 'IN'.

```

Comment:
Normally the c= line is now in an "m=" section (assuming "m=" section refers to the m- line and each associated attribute).